src/components/Card/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import Statistic from "../Statistic/Statistic"
+
+class Card extends React.Component {
+
+    constructor(props) {
```

Comment:
As we aren't doing anything except super I don't think we need a constructor here

Reply:
on it